Define templates in context (#1645)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/post_upgrade_template_checks.adoc

[[post_upgrade-updating-templates-parameters]]
= Updating Templates, Parameters, Lookup Keys and Values
During the upgrade process, {Project} attempts to locate macros that are deprecated for {Project} {ProjectVersion} and converts old syntax to new syntax for the default {Project} templates, parameters, and lookup keys and values.

> However, {Project} does not convert old syntax in the custom job or provisioning templates that you have created and in the cloned templates.

Reviewer: ```suggestion
However, {Project} does not convert old syntax in cloned templates and in custom job or provisioning templates that you have created.
```

{Project} 6.4 onwards uses the `redhat_register` snippet instead of the `subscription_manager_registration` snippet.

> If you upgrade from {Project} 6.3 and earlier, ensure to replace the `subscription_manager_registration` snippet in your custom job or provisioning templates as follows:

Reviewer: ```suggestion
If you upgrade from {Project} 6.3 and earlier, you must replace the `subscription_manager_registration` snippet in your custom job or provisioning templates as follows:
```
Unrelated to your PR but I think this would be a better word choice.

Author: I agree. Thanks!
